modules/apps/dynamic-data-mapping/dynamic-data-mapping-test-util/src/main/java/com/liferay/dynamic/data/mapping/test/util/DDMStructureTestUtil.java

```diff
@@ -277,6 +277,10 @@ public static String getSampleStructuredContent(
 			"en_US");
 	}
 
+	public static DDMStructure updateStructure(DDMStructure ddmStructure) {
+		return DDMStructureLocalServiceUtil.updateDDMStructure(ddmStructure);
+	}
+
```

Comment:
Since we will use `_ddmStructureLocalService`, you can delete this.

Reply:
Done